BillingApp/BillingInfo/templates/query_one.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
```

Comment:
I think we could just use the template `test_show_all` with giving different data